src/Command/LoadStoryCommand.php

```diff
@@ -0,0 +1,117 @@
+use Zenstruck\Foundry\Tests\Fixture\TestKernel;
+
+/**
+ * @author Nicolas PHILIPPE <nikophil@gmail.com>
+ */
+final class LoadStoryCommand extends Command
+{
+    public function __construct(
+        /** @var array<string, class-string<Story>> */
+        private readonly array $stories,
+        /** @var array<string, array<string, class-string<Story>>> */
+        private readonly array $groupedStories,
+        /** @var iterable<BeforeFirstTestResetter> */
+        private iterable $databaseResetters,
+        private KernelInterface $kernel,
+    )
+    {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->addArgument('name', InputArgument::OPTIONAL, 'The name of the story to load.')
+            ->addOption('append', 'a', InputOption::VALUE_NONE, 'Skip resetting database and append data to the existing database.')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        if (count($this->stories) === 0) {
+            throw new LogicException('No story as fixture available: add attribute #[AsFixture] to your story classes before running this command.');
+        }
+
+        $io = new SymfonyStyle($input, $output);
+
+        if (!$input->getOption('append')) {
+            $this->resetDatabase();
+        }
+
+        $stories = [];
+
+        if (null === ($name = $input->getArgument('name'))) {
+            $storyNames = array_keys($this->stories);
+            if (count($this->groupedStories) > 0) {
+                $storyNames[] = '(choose a group of stories...)';
+            }
+            $name = $io->choice('Choose a story to load:', $storyNames);
+
+            if (!isset($this->stories[$name])) {
+                $groupsNames = array_keys($this->groupedStories);
+                $name = $io->choice('Choose a group of stories:', $groupsNames);
+            }
+        }
+
+        if (isset($this->stories[$name])) {
+            $io->comment("Loading story with name \"{$name}\"...");
+            $stories = [$name => $this->stories[$name]];
+        }
+
+        if (isset($this->groupedStories[$name])) {
+            $io->comment("Loading stories group \"{$name}\"...");
+            $stories = $this->groupedStories[$name];
+        }
+
+        if (!$stories) {
+            throw new InvalidArgumentException("Story with name \"$name\" does not exist.");
+        }
+
+        foreach ($stories as $name => $storyClass) {
+            $storyClass::load();
+
+            if ($io->isVerbose()) {
+                $io->info("Story \"$storyClass\" loaded (name: $name).");
+            }
+        }
+
+        $io->success('Stories successfully loaded!');
+
+        return self::SUCCESS;
+    }
+
+    private function resetDatabase(): void
+    {
+        foreach ($this->databaseResetters as $databaseResetter) {
+            $databaseResetter->resetBeforeFirstTest($this->kernel);
+        }
+    }
```

Comment:
not sure what to do with global state here :thinking:

Reply:
Hmm, good question. You could certainly have one global state for tests, and another for dev but since most users probably have the same config for each, it could be unexpected to have it added by default.

I'm leaning to not including in the command right now: "global state is only for tests". In the future, if asked, we could add a `--load-global-state` option.

Reply:
skipping global state is actually more complex than I expected :shrug: 

everything is public (so that it could be decorated in userland), so I can't add a new param in the existing public methods :grimacing:

Reply:
ok, actually, we're calling `BeforeFirstTestResetter` which only loads global state if dama is enabled.

but dama should be disabled when running this command (otherwise, nothing will be persisted at the end of the command)... not so complex, then :slightly_smiling_face: